transform/func-lowering.go

```diff
@@ -55,17 +56,30 @@ func LowerFuncValues(mod llvm.Module) {
 	funcValueWithSignaturePtr := llvm.PointerType(mod.GetTypeByName("runtime.funcValueWithSignature"), 0)
```

Comment:
Sidenote: this will eventually need to be changed. I discovered that types may be renamed when LLVM modules are merged (when there are duplicates). This is a big problem in the interface lowering pass and I'm not yet sure how to fix that.